tallycash/extension
# Correctly display "To" address for ERC-20 Transactions

Closes #916 

This PR fixes the issue of the incorrect "To" address being shown on the Activity List for ERC-20 transfers, as well as a related issue of the wrong "To" address being shown in the Acitivity Details page as well.

### To Test
- Preview an address with some recent ERC-20 sends [(this is the one I used)](https://etherscan.io/address/0x136611b0040d9d5f1da33c78b49b3ab4ebfdb88f).
- Ensure that transactions where your address is sending ERC-20 tokens correctly show as `To` and show the correct recipient address in the Activity List
- When clicking into one of these activity items - ensure that the panel on top of the Activity Detail shows the correct `To` address.

<img width="384" alt="image" src="https://user-images.githubusercontent.com/94649004/159026321-41960f60-f213-422f-9245-02ed63fb9b74.png">

<img width="383" alt="image" src="https://user-images.githubusercontent.com/94649004/159027632-b7fd581b-4b72-430f-ac5b-1ae690cc5267.png">
